src/esp/metadata/attributes/AttributesBase.h

```diff
@@ -28,7 +28,7 @@ class AbstractAttributes : public esp::core::AbstractFileBasedManagedObject,
     AbstractAttributes::setHandle(handle);
   }
 
-  ~AbstractAttributes() override = default;
+  virtual ~AbstractAttributes() override = default;
```

Comment:
redundant. Surprise clang tidy didn't catch this.

Reply:
Is it? will inheriting destructors treat it as virtual? I was unclear.

Reply:
Heh. well clang-tidy agrees with you.  Groovy :)